### PR TITLE
Style core elements

### PR DESCRIPTION
**Pull request by prolajumokeoni**
## Awesome books: Style core elements
In this milestone we:
- used CSS to add styles to core elements

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -91,11 +94,13 @@ addButton.addEventListener('click', (e) => {
...
+    console.log(book.author);
     document.querySelector('#title').value = '';
     document.querySelector('#author').value = '';
   }
 });
 document.querySelector('.books').addEventListener('click', (e) => {
   Display.removeBookFromDisplay(e.target);
-  Storage.removeBook(e.target.previousElementSibling.textContent);
+  const myArr = e.target.previousElementSibling.previousElementSibling.textContent.trim().split(' ');
+  Storage.removeBook(myArr[myArr.length - 1]);
```
Comment (rloterh) [inline index.js:105]: - [ ] The `remove` function is not working as intended. When your `remove` button is triggered against a row, that row is deleted from the html code but when the page is refreshed the deleted object shows again. Kindly enhance this functionality by ensuring that targeted row is properly deleted from the `local Storage`

File: css/styles.css [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
-  margin-bottom: 5%;
+  width: 85%;
+  padding: 2% 0;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.books {
```
Comment (rloterh) [inline css/styles.css:24]: - [ ] The **added books** section/ table is not displaying book rows as `alternate row colors`. Kindly ensure to improve upon this aspect.  You can refer to [this resurce](https://www.geeksforgeeks.org/how-to-set-alternate-table-row-color-using-css/) to achieve similar styling in your design.

**Your design:**
image

**The wireframe design:**

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Muthomi & Priscilla, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Kindly check the comments marked **[OPTIONAL]** under the review:
- [OPTIONAL] 
You can improve the `Author` section of your ReadMe by replacing the text `Author1` with the actual full name of Author1
- [OPTIONAL] 
Your `live demo` is not displaying the current project. Kindly endeavor to update the same. 


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @prolajumokeoni** 

### **Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 


### **Optional**

- You can make the `.book: nth-child(odd) ` in you style little bit  lighter.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**